Tests/UnitTests/Plugins/Sycl/Seeding/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(ActsUnitTestSeedfinderSycl SeedfinderSyclTest.cpp CommandLineArguments.cpp)
+FIND_PACKAGE(Boost COMPONENTS program_options REQUIRED)
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIRS})
+TARGET_LINK_LIBRARIES(ActsUnitTestSeedfinderSycl PRIVATE ${unittest_extra_libraries} ${Boost_LIBRARIES})
```

Comment:
Ideally these 4 lines should rather just look like:

```cmake
add_executable(ActsUnitTestSeedfinderSycl
  SeedfinderSyclTest.cpp CommandLineArguments.h CommandLineArguments.cpp
  AtlasCuts.hpp SpacePoint.hpp)
target_link_libraries(ActsUnitTestSeedfinderSycl
  PRIVATE Boost::program_options ActsCore ActsPluginSycl)
```

I'm not a big fan of using `${unittest_extra_libraries}`, but my bigger point is that you should use the `Boost::program_options` imported target. Which is available to this configuration file even without calling `find_package(...)` in it explicitly.

Reply:
The `${unittest_extra_libraries}` are only needed if you are using the custom `add_unittest` macro to add the executables. You can do without here.

As mentioned above, the `find_package` calls are listed in a central location and should not be called again here.